.github/workflows/on_pull_request.yml

```diff
@@ -418,6 +418,11 @@ jobs:
         run: yarn test:website
         env:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
+          APPLITOOLS_BATCH_NAME: "${{ github.event.pull_request.head.repo.full_name }}: pull request from ${{ github.event.pull_request.head.ref }} - ${{ github.event.pull_request.title }} (${{ github.event.pull_request.head.sha }})"
```

Comment:
❗ This was pulled from the previous set up of Applitools with storybook. Need to:
1. look up the Github workflow integration with Applitols/cypress-eyes
2. confirm that this is the required batch name setup
3. Update as necessary (and maybeeee leave a comment/note)